components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/gen/java/org/wso2/carbon/apimgt/rest/api/core/dto/APIListDTO.java

```diff
@@ -0,0 +1,105 @@
+package org.wso2.carbon.apimgt.rest.api.core.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import org.wso2.carbon.apimgt.rest.api.core.dto.APIInfoDTO;
+import java.util.Objects;
+
+/**
+ * APIListDTO
```

Comment:
When we execute mvn clean install (with wso2 parent as 5), JavaDocs will fail if the javadoc for the class does not end with a full stop (".")